mindsdb/integrations/handlers/footballApi_handler/players_table.py

```diff
@@ -0,0 +1,134 @@
+import pandas as pd
+from mindsdb_sql.parser import ast
+
+from mindsdb.integrations.libs.api_handler import APITable
+from mindsdb.integrations.utilities.sql_utils import extract_comparison_conditions
+
+
+class PlayersTable(APITable):
+    def select(self, query: ast.Select) -> pd.DataFrame:
+        """Pulls data from the Football API
+
+                Parameters
+                ----------
+                query : ast.Select
+                   Given SQL SELECT query
+
+                Returns
+                -------
+                pd.DataFrame
+                    List of Football players matching the query
+
+                Raises
+                ------
+                ValueError
+                    If the query contains an unsupported condition
+                """
+        conditions = extract_comparison_conditions(query.where)
+        player_params = {}
+        for a_where in conditions:
+            key = a_where[1]
+
+            if a_where[0] != "=":
+                raise ValueError(f"Unsupported where operation for {key}")
+
+            # Assign values based on the key
+            if key == "league":
+                player_params["league"] = int(a_where[2])
+            elif key == "season":
+                player_params["season"] = int(a_where[2])
+            elif key == "team":
+                player_params["team"] = int(a_where[2])
+            elif key == "id":
+                player_params["id"] = int(a_where[2])
+            elif key == "search":
+                player_params["search"] = a_where[2]
+            elif key == "page":
+                player_params["page"] = int(a_where[2])
+            else:
+                raise ValueError(f"Unsupported where argument {key}")
+
+        player_data = self.handler.call_football_api("get_players", **player_params)
```

Comment:
I do not really understand the need to pass the method name to the `call_football_api()` function in the handler and then call that method from there. I think you can directly call the methods of the `PlayersApi` class here. It is fine to use the `call_football_api()` function, granted that you have a somewhat generic approach to call the Football API for any resource (players, teams etc.) and that it reduces the amount of code required for the implementation, but this does not really seem to do that. The use of the config (`footballapi_constants.py`) does not seem to be necessary either.